docs/source/examples/workspaces/dummy-aws/PinFile.dummy-aws.yml

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
why is dummy-aws in its own workspace while others like dummy-openstack are in random?

Reply:
good point. I'll fix that up.